libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -150,6 +150,27 @@ const AP_Param::GroupInfo AC_AttitudeControl::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("INPUT_TC", 20, AC_AttitudeControl, _input_tc, AC_ATTITUDE_CONTROL_INPUT_TC_DEFAULT),
 
+    // @Param: LAND_R_MULT
+    // @DisplayName: Roll gain multiplier used when landed
```

Comment:
I know a multiplier makes sense mathematically, but no-one is ever going to set it above 1 and so its really a divider. I would prefer a name that conveys to the use a bit more what it does.

Reply:
That would be a bit confusing. Unless we also invert the parameter...

Reply:
I think people understand that multiplying by a number between 0 and 1 will reduce the gains.